aegidii/__init__.py

```diff
@@ -0,0 +1,57 @@
+import json
+import re
+import os
+
+import parsel
+import requests
+
+
+class AegidiiBoothScraper:
+
+    MARKET_NAME = 'Aegidii'
+
+    def __init__(self):
+        urls_path = os.path.join(os.path.dirname(__file__), 'aegidii.json')
```

Comment:
The `aegidii.json` file in this PR is empty. Should it contain the list of urls you've listed in [this](https://github.com/codeformuenster/weihnachtsmarkt-data/pull/3#issuecomment-444219426) comment?

Reply:
It should contain ID->URL mappings once we have them, like
```
[
  {"id": 25, "url": "https://www.aegidii-weihnachtsmarkt.de/hortz"},
  # ...
]
```